apps/challenges/models.py

```diff
@@ -54,3 +87,22 @@ class ChallengePhase(TimeStampedModel):
     class Meta:
         app_label = 'challenges'
         db_table = 'challenge_phase'
+
+    def __str__(self):
+        """Returns the name of Phase"""
+        return self.name
+
+    def get_start_date(self):
+        """Returns the start date of Phase"""
+        return self.start_date
+
+    def get_end_date(self):
+        """Returns the end date of Challenge"""
+        return self.end_date
+
+    @property
+    def is_active(self):
+        """Returns if the challenge is active or not"""
+        if self.end_date > timezone.now():
+            return True
+        return False
```

Comment:
@deshraj: Can we add a check that end_date should always be greater than start_date? 
Also write a corresponding test case for that!